compiler/src/iree/compiler/Dialect/VM/Target/Bytecode/test/constant_encoding.mlir

```diff
@@ -121,6 +122,32 @@ vm.module @constants {
   // CHECK-NEXT:   128,
   // CHECK-NEXT:   63
   // CHECK-NEXT: ]
-  vm.rodata private @splat_float32 dense<1.000000e+00> : tensor<3xf32>
+  vm.rodata private @splat_f32 dense<1.000000e+00> : tensor<3xf32>
 
+  // Tests that elided tensors of sub-byte types get filled with zeros when the
+  // --iree-util-zero-fill-elided-attrs flag is passed. This is useful for
+  // testing compilation without wanting to keep the large constants around
+  // during iteration.
+  //      CHECK: "embedded_data": [
+  // CHECK-NEXT:   0,
+  // CHECK-NEXT:   0,
+  // CHECK-NEXT:   0
+  // CHECK-NEXT: ]
+  vm.rodata private @elided_i2 dense_resource<__elided__> : tensor<9xi2>
+
+  //      CHECK: "embedded_data": [
+  // CHECK-NEXT:   0,
```

Comment:
I wonder if we can use `CHECK-COUNT-k: 0,` here

Reply:
tricky due to JSON - the trailing element has no comma so it'd have to be a CHECK-COUNT-(k-1) + CHECK-NEXT ;(